docker/client/deploy.tf

```diff
@@ -8,8 +8,9 @@ resource "null_resource" "configure-docker-client-certs" {
     docker_client_certs_pem   = "${element(tls_locally_signed_cert.docker_client.*.cert_pem, count.index)}"
     validity_period_hours     = "${var.validity_period_hours}"
     early_renewal_hours       = "${var.early_renewal_hours}"
-    ip_addresses_list         = "${var.ip_addresses_list}"
-    dns_names_list            = "${var.dns_names_list}"
+    ip_addresses_list         = "${join(",",var.ip_addresses_list)}"
```

Comment:
I can see why I added in the first time https://github.com/Capgemini/tf_tls/pull/38/commits/b1a4420f3f31bfb124fdc172a9749a69dbbf93dd not sure if this still apply for Terraform v0.7.13
Is this pr actually any different from https://github.com/Capgemini/tf_tls/pull/38/ ?